packages/suite/src/actions/wallet/coinmarketCommonActions.ts

```diff
@@ -91,3 +94,27 @@ export const verifyAddress = (path: string, address: string) => async (
         );
     }
 };
+
+export const submitRequestForm = (tradeForm: BuyTradeFormResponse) => async (
+    dispatch: Dispatch,
+    getState: GetState,
+) => {
+    const { device } = getState().suite;
+    if (device && !device.remember) {
+        dispatch(suiteActions.setFlag('forcedRememberDevice', true));
+        dispatch(suiteActions.rememberDevice(device));
+    }
+    utilsSubmitRequestForm(tradeForm);
+};
+
+export const forgetForcedRememberDevice = () => (dispatch: Dispatch, getState: GetState) => {
+    const { forcedRememberDevice } = getState().suite.flags;
```

Comment:
if you keep `device.state` in forcedRememberDevice field then here you should be able forget desired device. 
we need to be careful here, there might be potential race condition with two devices connected
`const device = getState().devices.find(d => d.state === forcedRememberDevice)`